lib/trento_web/templates/layout/live.html.heex

```diff
@@ -1,11 +1,11 @@
 <main class="container">
-  <p class="alert alert-info" role="alert"
-    phx-click="lv:clear-flash"
-    phx-value-key="info"><%= live_flash(@flash, :info) %></p>
+  <p class="alert alert-info" role="alert" phx-click="lv:clear-flash" phx-value-key="info">
+    <%= Phoenix.Flash.get(@flash, :info) %>
```

Comment:
I think simply `get` instead of `Phoenix.Flash.get` would work as well, as we are importing `Phoenix.Component` in the router.
Anyway, nothing bad having explicit :+1: